### PR TITLE
chore: 3.1 news

### LINKED ISSUE #1177: docs: news for 3.1
**Please describe the feature**

As a user, I want the news to reflect the state of the project as released/deployed.

**Acceptance Criteria**

- [x] updated news for the upcoming 3.1 release
- [x] mention of the posters shown at ECCB and ICSB

### PR DESCRIPTION
**Pull request by mihai-sysbio**
**Related issue(s) and PR(s)**  
This PR closes #1177.

News 3.1 news:
> The _Interaction Partners_ has been redone, and it now shares the same underlying solution as the _3D Map Viewer_. With this, the library used for rendering networks has also been updated and improved. Moreover, the _Data Overlay_, which is shared between the _Interaction Partners_ and the _Map Viewer_, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. These data types have also been renamed to better reflect their usage.
Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well.

The redo of the _Interaction Partners_ was carried out mainly through issues #1130 and #1144. There is more work, however, implemented in the https://github.com/MetabolicAtlas/3d-network-viewer repository.
The integrated models were updated through #1081.

_Data overlay_:
- #1066
- #1121
- #1181

Bug fixes:
- #848
- #968
- #1055
- #1074
- #1124 

Improvements of the user experience and documentation:
- #914
- #931
- #1017
- #1075
- #1096
- #1097
- #1110
- #1112
- #1113
- #1150

Tech related issues:
- #867
- #917
- #962
- #1048
- #1093
- #1116
- #1157
- #1073
- #1080
- #1089

### CODE REVIEW

**Comment by inghylt** [comment]
I think it looks good! Just a though, we should be able to merge #1152  and #1181 soon and I don't think we would need to update the news text, just add them onto the list of issues. But we could also wait and have them in a future release, just wanted to raise the option

File: frontend/src/content/news.json [hunk 1]
```diff
@@ -1,9 +1,24 @@
 {
   "2022": [
+    {
+      "date": "2022-10-31",
+      "title": "Release 3.1 improves Interaction Partners and Data Overlay",
+      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, now shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```
Comment (e0) [inline frontend/src/content/news.json:6]: Not sure if it's just me but this part: ", now shared between the <i>Interaction Partners</i> and the <i>Map Viewer</I>," while technically true, gives me the impression that this was added in `3.1`. I think this was actually first introduced in this PR: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/723. So I suggest that we can either remove the `now` or the entire quoted part.

```suggestion
      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

**Review by e0 — APPROVED** [review]
Added a comment but it's not a big deal so I can approve it.

File: frontend/src/content/news.json (hunk 1 above)
Comment (mihai-sysbio) [reply frontend/src/content/news.json:6]: I'd say it's more debatable - even though is was shared, now we have the reaction overlay which I don't think would have been part of the IP data overlay if it weren't for this work. Anyway, it's safer to remove the `now`,  but I'd go with another suggestion:

```suggestion
      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, which is shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

**Review by MalinAhlberg — APPROVED** [review]
Looks good :rocket: !

File: frontend/src/content/news.json (hunk 1 above)
Comment (e0) [reply frontend/src/content/news.json:6]: I like the new suggestion!

**Comment by nanjiangshu** [comment]
If #1181 is going to be merged before release 3.1, I think it should be reflected in the text.  Otherwise, it looks good to me.

**Comment by mihai-sysbio** [comment]
> If #1181 is going to be merged before release 3.1, I think it should be reflected in the text. Otherwise, it looks good to me.

It is now listed in the grouped issues in the PR body. Or are you thinking of the news text?

**Comment by nanjiangshu** [comment]
> > If #1181 is going to be merged before release 3.1, I think it should be reflected in the text. Otherwise, it looks good to me.
> 
> It is now listed in the grouped issues in the PR body. Or are you thinking of the news text?
I mean text in news. Something like "The data types in Data overlay have been renamed to better reflect their biological nature" or simply "... so that they are more understandable"

File: frontend/src/content/news.json [hunk 2]
```diff
@@ -1,9 +1,24 @@
 {
   "2022": [
+    {
+      "date": "2022-10-31",
+      "title": "Release 3.1 improves Interaction Partners and Data Overlay",
+      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, which is shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```
Comment (mihai-sysbio) [inline frontend/src/content/news.json:6]: @nanjiangshu here is the implemented suggestion:
```suggestion
      "text": "<b>Metabolic Atlas 3.1</b> is out. The <i>Interaction Partners</i> has been redone, and it now shares the same underlying solution as the <i>3D Map Viewer</i>. With this, the library used for rendering networks has also been updated and improved. Moreover, the <i>Data Overlay</i>, which is shared between the <i>Interaction Partners</i> and the <i>Map Viewer</i>, has been extended to support reaction data overlay, in addition to multiple simultaneous data overlays of different types. These data types have also been renamed to better reflect their usage. <br> Numerous small bugs have been resolved, and the software stack has seen several major updates. Finally, the integrated models have been updated as well. <br> More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/3.1' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
```

Comment (nanjiangshu) [reply frontend/src/content/news.json:6]: Looks good.

**Pull request merged**